authorization-for-rag/langchain-next-js/src/lib/actions/documents.ts

```diff
@@ -37,7 +37,7 @@ export const createDocument = async (input: NewDocumentParams, text: string) =>
     );
 
     // write the relationship tuples to FGA
-    await addRelation(user.email, document.id);
+    await addRelation(user.sub, document.id);
```

Comment:
The feature is meant for share by email and if sub is not same as email then this will break, can you please revert this change back

Reply:
If email is preferred, seems we could preserve the `getUser()` call in the passthrough that was here before to preserve it in the `body` then. I dont believe it is passed on the AT by default. cc: @PavelKorobchuk  

```
export const { GET, POST, PUT, PATCH, DELETE, OPTIONS, runtime } = initApiPassthrough({
  apiUrl: process.env.LANGGRAPH_API_URL,
  baseRoute: 'chat/',
  bodyParameters: async (req, body) => {
    if (req.nextUrl.pathname.endsWith('/runs/stream') && req.method === 'POST') {
      return {
        ...body,
        config: {
          configurable: {
            _credentials: {
              user: await getUser(),
            },
          },
        },
      };
    }

    return body;
  },
```